src/main/java/de/hpi/bpt/argos/common/validation/RestInputValidationServiceImpl.java

```diff
@@ -26,7 +26,8 @@ public int validateInteger(String inputValue, Function<Integer, Boolean> validat
 				throw new InputValidationException(inputValue, "Integer");
 			}
 		} catch (InputValidationException e) {
-			logger.error(e.getMessage(), e);
+			logger.error(e.getMessage());
+			logger.trace("Reason: ", e);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Reason: " 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)